src/__test__/components/plots-and-tables/Volcano.test.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import preloadAll from 'jest-next-dynamic';
+import { Provider } from 'react-redux';
+import configureMockStore from 'redux-mock-store';
+import '@testing-library/jest-dom';
+import thunk from 'redux-thunk';
+import _ from 'lodash';
+import VolcanoPlot from '../../../pages/experiments/[experimentId]/plots-and-tables/volcano/index';
+import {
+  initialPlotConfigStates,
+} from '../../../redux/reducers/componentConfig/initialState';
+
+const mockStore = configureMockStore([thunk]);
+const experimentId = 'insaneExperiment';
+let store;
```

Comment:
Why do you need to initialize `store` here? Can't you just initialize `const store` inside `renderVolcano`?